src/Microsoft.DotNet.Darc/src/Darc/Operations/AddChannelOperation.cs

```diff
@@ -42,7 +43,24 @@ public override async Task<int> ExecuteAsync()
                 }
 
                 Channel newChannelInfo = await remote.CreateChannelAsync(_options.Name, _options.Classification);
-                Console.WriteLine($"Successfully created new channel with name '{_options.Name}'.");
+                switch (_options.OutputFormat)
+                {
+                    case DarcOutputType.json:
+                        Console.WriteLine(JsonConvert.SerializeObject(
+                            new
+                            {
+                                id = newChannelInfo.Id,
+                                name = newChannelInfo.Name,
+                                classification = newChannelInfo.Classification
+                            },
+                            Formatting.Indented));
+                        break;
+                    case DarcOutputType.yaml:
+                        Console.WriteLine($"Successfully created new channel with name '{_options.Name}' and id {newChannelInfo.Id}.");
```

Comment:
This isn't really yaml...  Is OutputType.Text a thing? Also... why are these enumerations lower cased?  Did they already exist that way?

Reply:
Because the command line parser wants to do case sensitive parsing.  I can change yaml->text, which makes more sense. There is a mix of yaml and text sitting in darc by default. See the discussion of the json issue in https://github.com/dotnet/arcade/issues/2099#issuecomment-584715852